tests/test-sources/plugins/by-name/wrapping/default.nix

```
@@ -0,0 +1,11 @@
+{ pkgs, ... }:
+{
+  empty = {
+    plugins.wrapping.enable = true;
+  };
+
+  defaults.plugins.which-key = {
+    enable = true;
+    notify_on_switch = true;
```

```suggestion
    settings = {
      notify_on_switch = true;
    };
```
This option will be created within the `settings` option.